src/azure-cli/azure/cli/command_modules/acr/build.py

```diff
@@ -97,7 +97,7 @@ def acr_build(cmd,  # pylint: disable=too-many-locals
 
     platform_os, platform_arch, platform_variant = get_validate_platform(cmd, platform)
 
-    DockerBuildRequest, PlatformProperties = cmd.get_models('DockerBuildRequest', 'PlatformProperties')
+    DockerBuildRequest, PlatformProperties = cmd.get_models('DockerBuildRequest', 'PlatformProperties', operation_group='runs')
```

Comment:
How do you determine the operation group name? Is there a list that track what modules included in each group?

Reply:
It's defined in [_shared.py](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli-core/azure/cli/core/profiles/_shared.py)

Reply:
check the file src\azure-cli-core\azure\cli\core\profiles\_shared.py
Pretty much the default version (when not specified) is 2020-11-01-preview'
and all the other defined groups in the file are referring the same API version:
'agent_pools': '2019-06-01-preview',
'tasks': '2019-06-01-preview',
'task_runs': '2019-06-01-preview',
'runs': '2019-06-01-preview',